### DISCUSSION TITLE
Allow deleting/writing the same tuple key in the same Write call

### DISCUSSION THREAD

**Discussion by aaguiarz** [openfga/community#282] [Ideas]
### Checklist

- [X] I agree to the terms within the [OpenFGA Code of Conduct](https://github.com/openfga/.github/blob/main/CODE_OF_CONDUCT.md).

### Describe the problem you'd like to have solved

If we want to change the context values of a tuple, I need to delete/write the tuple in the same write() call.

However, OpenFGA tries to write it first, and it returns an error as it's duplicated.

### Describe the ideal solution

We should first process deletes and then writes.

### Alternatives and current workarounds

Make 2 write calls in different transactions.

**Comment by cristianbriscaru** [comment]
Hi @aaguiarz , any updates on this feature? I think this is an absolute must for openFGA in order to be able to properly update tuples
